src/components/dialogs/filter/expert/expert-filter-constants.ts

```diff
@@ -33,27 +33,60 @@ export const ENERGY_SOURCE_OPTIONS = [
     { name: 'OTHER', label: 'Other' },
 ];
 
+// customName is used to export to the server
 export const OPERATOR_OPTIONS = {
-    EQUALS: { name: OperatorType.EQUALS, label: 'equality' },
-    NOT_EQUALS: { name: OperatorType.NOT_EQUALS, label: 'notEquality' },
+    EQUALS: { name: '=', customName: OperatorType.EQUALS, label: 'equality' },
+    NOT_EQUALS: {
+        name: '!=',
+        customName: OperatorType.NOT_EQUALS,
+        label: 'notEquality',
+    },
     // Number and String
-    EXISTS: { name: OperatorType.EXISTS, label: 'exists' },
+    EXISTS: {
+        name: OperatorType.EXISTS,
```

Comment:
name: 'exists',

Reply:
It's on purpose, 'exists' is not a RQB operator

Reply:
Yes